exercicios/para-casa/dataset_alunas.py

```diff
@@ -59,4 +59,129 @@
         "Presença": [True, True, True, False, True],
         "Participação": 7.5
     }
-}
\ No newline at end of file
+}
+
+
+
+print ("-------------------------hello world!!---------------------------------")
+print ("-------------------Esse é o sistema reprograma-------------------------")
+print ("-----------------------seja bem vinde ;) ------------------------------")
+print ("----------Para ter acesso a lista de alunas (digite 1)-----------------")
+print ("----------------------Para inserir aluna nova (digite 2)--------------")
+print ("-------------------Para consultar presença (digite 3)-----------------")
+print ("----------------Para consultar notas  (digite 4)------------------------")
+print ("----------------Para calcular média (digite 5)------------------------")
+print ("-----------------Para sair do sistema (digite 6)----------------------")
+
+ 
+
+ 
+
+     
+    
+def inclusao_novas_alunas ():
+    while True:
+     nome = str(input("Insira o nome: "))
+     sobrenome = input("Insira o seu sobrenome: ")
+     turma = input(" Insira a sua turma: ")
+   
+     notas = []
+     for i in range (3):
+       nota = int(input (f"Insira as suas {i+1} notas: "))
+       notas.append(nota)
+       
+     presenca = []
+     for i in range (5):
+       aulas = bool(input(f"insira presença {i+1} com 'True' e falta com 'False' são 5 aulas: ")) # returnar, criar função bool
+       presenca.append(presenca)
```

Comment:
A linha 96 está tentando inserir a lista chamada presenca dentro da lista chamada presenca. O dado que queremos inserir na lista presenca é aulas ne? :)

Sobre utilizar o bool, o código está tentando converter a entrada do usuário para um valor booleano. Em Python, bool() converte uma string não vazia para True, independentemente do conteúdo da string. Por exemplo:
entrada = bool(input("Digite algo: "))

Se o usuário digitar qualquer coisa, como "True", "False", "Sim", "Não" ou qualquer palavra, bool() vai considerar isso como True.

**Sugestão:**
```suggestion
    while True:
        entrada = input(f"Insira presença {i+1} com 'True' ou falta com 'False' são 5 aulas: ")
        if entrada == 'True':
            presenca.append(True)
            break
        elif entrada == 'False':
            presenca.append(False)
            break
        else:
            print("Entrada inválida. Por favor, insira apenas 'True' ou 'False'.")
```